src/components/Restaurant.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Order from "./Order";
+
+const Restaurant = ({ orderType }) => {
+  // const [orders,setOrders] =useState(0);
```

Comment:
Remember to delete old comments as you're going along @leilafaez. If you need to review some old code that was deleted you can always checkout the code at an earlier version if need be using Git.